drivers/src/uart.rs

```diff
@@ -46,13 +46,30 @@ impl Uart {
     /// # Arguments
     ///
     /// `byte` - Byte to write to UART
+    #[cfg(not(feature = "fpga_realtime"))]
     pub fn write_byte(&mut self, byte: u8) {
         // TODO: cleanup after final UART RTL definition is in place
         const STDOUT: *mut u32 = 0x3003_00C8 as *mut u32;
         unsafe {
             ptr::write_volatile(STDOUT, byte as u32);
         }
     }
+
+    #[cfg(feature = "fpga_realtime")]
+    pub fn write_byte(&mut self, byte: u8) {
```

Comment:
This is fine for now, but I do have an idea for future unification:

1. Change write_byte() to toggle a "char_clk" pin in generic_output_wires, which is edge-triggered to signify that a new byte  has been written.
2. Add a character FIFO (4k?) to the FPGA that buffers up the written bytes for later access by the A53 cpu.

Reply:
Added to my TODO list. Marking as resolved to move on with merge.